lib/applications/resolvers.ts

```diff
@@ -547,3 +549,93 @@ export const createReplacementApplication: Resolver = async (_, args, { prisma }
     ok: true,
   };
 };
+
+export const generateApplicantsReport: Resolver = async (_, args, { prisma }) => {
+  const {
+    input: { startDate, endDate, columns },
+  } = args;
+
+  const columnsSet = new Set(columns);
+
+  const applications: GenerateApplicantsReportApplications[] = await prisma.application.findMany({
```

Comment:
TypeScript was throwing errors when I tried adding `totalAmount`, `applicantName` and `rcdPermitId` properties to the `applications` object (https://github.com/uwblueprint/richmond-centre-for-disability/blob/jb/csv-generation-api-requests/lib/applications/resolvers.ts#L589-L599) so I had to create a new type (`GenerateApplicantsReportApplications`) with those properties and explicitly set `applications` to that type